C-Sharp.md

```diff
@@ -1,6 +1,268 @@
-#The C# Programming Language
+# The C# Programming Language
 
+1. ### Basics
+  * Setup
+  * Your first C# program
```

Comment:
Do you mean capitalize each item of ordered list ?
like 
* BASICS
* INTERMEDIATE